src/olympia/devhub/tests/test_views_validation.py

```diff
@@ -151,62 +151,79 @@ def test_results_page(self):
     def test_only_dev_can_see_results(self):
         self.client.logout()
         assert self.client.login(email='regular@mozilla.com')
-        assert self.client.head(self.url, follow=True).status_code == 403
+        assert self.client.head(self.url, follow=False).status_code == 403
```

Comment:
What are all the switches from `follow=True` to `follow=False` about?

Reply:
Those `follow=True` force the test client to follow redirects, but in this particular case there is no good reason to do so as it would just hide similar class of bugs. If you look at the tests I've added, but set them to follow redirect like the ones that I modified, then the added tests pass even without any chances, since what we're concerned about here *is* an erroneous redirect.